src/js/lib/youtube.js

```diff
@@ -21,11 +21,13 @@ export function pimpYouTubePlayer(videoId, node, height, width, chapters) {
 
                             const playerTotalTime = youTubePlayer.getDuration();
                             chapTimer = setInterval(function() {
+                                let chapterCurrentProgress;
```

Comment:
> Is there another place I could declare it?

I'd move `chapterCurrentProgress = (playerCurrentTime-chapStart)/(chapNext-chapStart);` out from the `filter` method and do this work inside the `forEach` later on.

I'd say `filter` should only be responsible for finding a value in an array make things easier to reason about.